ExScripts-NickTracker/ExtioNickTrackerVer5.6.kvs

```diff
@@ -0,0 +1,599 @@
+					}
+					if ($4==off)
+					{
+						# echo Previous power state: $config.read(%NTSet,NTpower,"on") ;
+						config.write %NTSet NTdisplayUser "off" ;
+					}
+					echo NickTracker user tracking state now: $config.read(%NTSet,NTdisplayUser,"on") ;
+				};
+			};
+			if ($2==display)
+			{
+				if ($3==ip)
+				{				
+					if ($4==on)
+					{
+						# echo Previous power state: $config.read(%NTSet,NTpower,"on") ;
+						config.write %NTSet NTdisplayIp "on" ;
+					}
+					if ($4==off)
+					{
+						# echo Previous power state: $config.read(%NTSet,NTpower,"on") ;
+						config.write %NTSet NTdisplayIp "off" ;
+					}
+					echo NickTracker Ip Display state now: $config.read(%NTSet,NTdisplayIp,"on") ;
+				};
+			};
+			if ($2==nicknumber)
+			{
+				if ($isNumeric($3) && $3 > 0)
+				{
+					echo Previous Number of nicks: $config.read(%NTSet,NickNum,6) ;
+					config.write %NTSet NickNum $3 ;
+					echo New Number of nicks to show: $config.read(%NTSet,NickNum,6) ;
+					echo warning: If the new number is lower than the old, you will lose the
+					echo previous nicks stored above the number of the new number when processed.
+				} else {
+					dialog.textinput -d=6 (NickNumber Setting,"<center>What do you want to set the<br>number of saved nicks to?</center>","Set","Cancel")
+					{
+						switch($0)
+						{
+							case(0):
+							{ if($isNumeric($1)) { nicktracker set nicktracker nicknumber $1; } else echo you need to use a number for that setting. canceled.; }
+							break;
+							default:
+							# Cancelled
+							break;
+						}
+						echo John 3:16 says it simply.  God loves us, and died for us.  We would be fools to pass up what he gives.;
+					};
+				};
+			};
+		};
+		config.close %NTSet ;
+	}
+	if ($0 == "identify") 
+	{
+		if ($1)
+		{
+			%NT = $config.open(NickTracker.kvs) ;
+			%NT2 = $config.open(NickTracker2.kvs) ;
+#			%identifyNick[] = $str.split([!@],$1,r) ;
+			%identifyNick[] = $str.split([!@],$mask($1,0),r) ;
+			NT::NickTrackerProcess %identifyNick[0] %identifyNick[1] %identifyNick[2] 4 ;
+			config.close %NT ;
+			config.close %NT2 ;
+		}
+	}
+	if ($0 == "channel") 
+	{
+		%NT = $config.open(NickTracker.kvs) ;
+		%NT2 = $config.open(NickTracker2.kvs) ;
+		if ($1) 
+		{
+			if ($str.contains($1,"#"))
+			{
+				if ($channel($1) != 0) { echo Showing NickTracker for $1; %chanUsersArray[] = $chan.users($channel($1),,m);	} else { echo Channel does not exist.; };
+			} else { echo channel specified needs a pounds sign.; } ;
+		} else { if ($1 != "") { echo Current channel:; %chanUsersArray[] = $chan.users($channel($chan.name),,m); } else { echo Non-Channel passed, please select a valid channel ;}; };
+		%j = 0 ;
+		# echo chanUsersArray: %chanUsersArray[] ;
+		while (%j < %chanUsersArray[]#)
+		{
+			%chanUsersArrayPieces[] = $str.split([!@],%chanUsersArray[%j],r) ;
+			NT::NickTrackerProcess %chanUsersArrayPieces[0] %chanUsersArrayPieces[1] %chanUsersArrayPieces[2] 4 ;
+			%j++ ;
+		} ;
```

Comment:
I dont think you need any of these semi colons after end of curly brackets in kvs, but if you're going to have them for whatever reason maybe they should not have spaces in between curly bracket and semi colon.